script.js

```diff
@@ -1,49 +1,64 @@
-let msgWaiting = document.querySelector(".message-waiting");
-let searchForm = document.getElementById("search-form");
-
+const msgWaiting = document.querySelector(".message-waiting");
+const searchForm = document.getElementById("search-form");
 const booksDisplay = document.getElementById("books-display");
 
-let getGoogleBooks = "https://www.googleapis.com/books/v1/volumes?q=";
-
-let apiKey = "&key=AIzaSyCw6aeDwd2srAlaiVBFSSBvyl2O7atCTaA";
+const googleBooksURL = "https://www.googleapis.com/books/v1/volumes?q=";
+const apiKey = "&key=AIzaSyCw6aeDwd2srAlaiVBFSSBvyl2O7atCTaA";
```

Comment:
Exposing an API key to the public can be dangerous. You sometimes hear stories of API keys being stolen & used extensively, leading to giant unexpected (and unwanted) usage bills.

It's not very easy hiding a key when you're working with client-side code, though. Even if you hid the key from GitHub (for example by using  [environment variables](https://stackoverflow.com/questions/53648652/how-to-use-environment-variables-in-github-page)), your API key would be exposed in the developer tools – anyone could in theory look at the network tab to find it.

So, when we're thinking about security, the question isn't how to we eliminate risk, but how much risk are we comfortable with? And what can we do to lower it to an acceptable level?

I recommend you [set up HTTP restrictions](https://cloud.google.com/docs/authentication/api-keys#api_key_restrictions) on the Google API side. You should only allow requests using this API key from your GitHub pages URL once your site is deployed. (For ease of use, during development you can remove those restrictions but it's strongly recommended you turn them on once you've deployed your work).